www/docs/en/dev/platform_plugin_versioning_ref/index.md

```diff
@@ -110,25 +118,25 @@ Some Examples:
 
 * **`cordova platform update /path/to/android/platform --save`**
```

Comment:
--save necessary here?

Reply:
I would say I can remove this. 

I don't remember adding `--save` so it might have been there from before or copy paste. 

Anyways, `cordova platform --help` also says nothing about `--save`, so I will remove.